src/components/DeferLoad/index.ts

```diff
@@ -67,22 +67,6 @@ class DeferLoad extends BridgeBase {
 
     // NOTE: Need to bind the callback to 'this' so it has this element's context when it's called.
     this.registerAndSubscribeComponent(this.onStateChange.bind(this));
-
-    PCore.getPubSubUtils().subscribe(
-      PCore.getConstants().PUB_SUB_EVENTS.EVENT_CANCEL,
```

Comment:
Do we no longer need to respond to these events?  I don't see them added ack anywhere in the refactoring so I would like to understand what's happening more.